### PR TITLE
swallow allowed_values error

### PR DESCRIPTION
**Pull request by seanpearsonuk**

### CODE REVIEW

**Review by mkundu1 — APPROVED** [review]

**Comment by seanpearsonuk** [comment]
@h-krishnan @gyeole I've tried this successfully for the linked bug. vol report def expr_list has no allowed-values despite being a stringlist. Is it in the contract for all strings/stringlists to have that attribute, such that the framework adds an empty one it if missing, or is the assumption wrong in flobject?

**Comment by seanpearsonuk** [comment]
@jharming

**Review by prmukherj — APPROVED** [review]

**Comment by seanpearsonuk** [comment]
@mkundu1 @jharming we will wait before merging this. The Fluent fix is the one we really want.

File: src/ansys/fluent/core/solver/flobject.py [hunk 1]
```diff
@@ -212,7 +212,10 @@ class Textual(Property):
 
     def allowed_values(self):
         """Get the allowed values of the object."""
-        return self.get_attr("allowed-values", (list, str))
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/solver/flobject.py:215]: @prmukherj regarding this old PR, this could be relevant to the testing issue task #1496.

Comment (seanpearsonuk) [reply src/ansys/fluent/core/solver/flobject.py:215]: @prmukherj @mkundu1 @hpohekar @ajain-work this change has now been superseded by identical changes - closing PR

**Pull request closed without merging**